test/src/main/java/org/apache/accumulo/test/TableOperationsIT.java

```diff
@@ -688,67 +688,68 @@ private void verifyTablesWithSplits(String tableName, Map<String,String> idMap,
 
     // verify individual tablets can be retrieved
     expectedTabletAvailability.clear();
     setExpectedTabletAvailability(expectedTabletAvailability, tableId, splitPts[0], null,
         tabletAvailability);
-    verifyTabletAvailabilites(tableName, new Range(null, new Text(splitPts[0])),
+    verifyTabletAvailabilities(tableName, new Range(null, new Text(splitPts[0])),
         expectedTabletAvailability);
 
     expectedTabletAvailability.clear();
     setExpectedTabletAvailability(expectedTabletAvailability, tableId, splitPts[1], splitPts[0],
         tabletAvailability);
-    verifyTabletAvailabilites(tableName,
+    verifyTabletAvailabilities(tableName,
         new Range(new Text(splitPts[0]), false, new Text(splitPts[1]), true),
         expectedTabletAvailability);
 
     expectedTabletAvailability.clear();
     setExpectedTabletAvailability(expectedTabletAvailability, tableId, splitPts[2], splitPts[1],
         tabletAvailability);
-    verifyTabletAvailabilites(tableName,
+    verifyTabletAvailabilities(tableName,
         new Range(new Text(splitPts[1]), false, new Text(splitPts[2]), true),
         expectedTabletAvailability);
 
     expectedTabletAvailability.clear();
     setExpectedTabletAvailability(expectedTabletAvailability, tableId, null, splitPts[2],
         tabletAvailability);
-    verifyTabletAvailabilites(tableName, new Range(new Text(splitPts[2]), false, null, true),
+    verifyTabletAvailabilities(tableName, new Range(new Text(splitPts[2]), false, null, true),
         expectedTabletAvailability);
 
     expectedTabletAvailability.clear();
     setExpectedTabletAvailability(expectedTabletAvailability, tableId, splitPts[1], splitPts[0],
         tabletAvailability);
     setExpectedTabletAvailability(expectedTabletAvailability, tableId, splitPts[2], splitPts[1],
         tabletAvailability);
-    verifyTabletAvailabilites(tableName,
+    verifyTabletAvailabilities(tableName,
         new Range(new Text(splitPts[0]), false, new Text(splitPts[2]), true),
         expectedTabletAvailability);
   }
 
-  public static void verifyTabletAvailabilites(String tableName, Range range,
-      List<AvailabilityForTablet> expectedAvailability) throws TableNotFoundException {
-    verifyTabletAvailabilites(accumuloClient, tableName, range, expectedAvailability);
+  public static void verifyTabletAvailabilities(String tableName, Range range,
+      Map<TabletId,TabletAvailability> expectedAvailability) throws TableNotFoundException {
+    verifyTabletAvailabilities(accumuloClient, tableName, range, expectedAvailability);
   }
 
-  public static void verifyTabletAvailabilites(AccumuloClient client, String tableName, Range range,
-      List<AvailabilityForTablet> expectedAvailability) throws TableNotFoundException {
+  public static void verifyTabletAvailabilities(AccumuloClient client, String tableName,
+      Range range, Map<TabletId,TabletAvailability> expectedAvailability)
+      throws TableNotFoundException {
```

Comment:
With this new map, may be able to simplify the body of this method to the following.

```java
    Map<TabletId, TabletAvailability> seenAvailability = client.tableOperations()
            .getTabletInformation(tableName, range).collect(Collectors.toMap(TabletInformation::getTabletId, TabletInformation::getTabletAvailability));
    assertEquals(expectedAvailability, seenAvailability);
```

Reply:
Makes sense, updated.